packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -43,15 +41,28 @@ const styles = (theme: ThemeType): JssStyles => ({
     maxWidth: 252,
   },
   message: {
+    display: "flex",
+    alignItems: "flex-start",
     fontSize: 18,
-    marginBottom: 18,
+    lineHeight: 1.75,
+  },
+  mailIcon: {
+    marginTop: 4,
+    marginRight: 12
+  },
+  checkIcon: {
+    color: "#4caf50",
+    marginTop: 4,
+    marginRight: 12
   },
   emailInput: {
+    marginTop: 18
   },
   subscribeButton: {
-    margin: "0 auto",
+    margin: "18px auto 0",
     display: "block",
-    background: "#d2e8d2",
```

Comment:
Check this looks ok on LW

Reply:
Their `theme.palette.primary.main` is a forest-y green:

<img width="500" alt="Screen Shot 2021-09-09 at 9 12 21 PM" src="https://user-images.githubusercontent.com/9057804/132755904-04eb47ef-3f42-4d99-9409-3c4482b32fc7.png">